Cython/Includes/libcpp/complex.pxd

```diff
@@ -8,21 +8,28 @@ cdef extern from "<complex>" namespace "std" nogil:
         # How to make the converting constructor, i.e. convert complex[double]
         # to complex[float]?
 
-        complex[T] operator+(complex[T]&)
-        complex[T] operator-(complex[T]&)
+        #complex[T] operator+(complex[T]&)
+        #complex[T] operator-(complex[T]&)
```

Comment:
Left here for comments: those lines are preventing the `cythonization` of valid code

Reply:
I think you can remove these lines

Reply:
Done!